lib/action_subscriber/babou.rb

```diff
@@ -0,0 +1,103 @@
+module ActionSubscriber
+  module Babou
+    ##
+    # Class Methods
+    #
+    def self.configure_from_yaml_and_cli(cli_options = {})
+      env = ENV["RAILS_ENV"] || ENV["RACK_ENV"] || ENV["APP_ENV"] || "development"
+
+      yaml_config = {}
+      babou_absolute_config_path = ::File.expand_path(::File.join("config", "babou.yml"))
```

Comment:
Does anyone else think it would be worth searching for an `action_subscriber.yml` file and falling back to `babou.yml` if we don't find one?  I'm thinking that supporting `babou.yml` is nice for backwards compatibility, but we should be sticking with the ActionSubscriber name for new users.